### ISSUE TITLE
UUID Disk Identification

### ISSUE THREAD

**Issue by Baitinq** [nix-community/disko#14]
I see that disks right now in the config are identified by their given name (e.g. sda), would it be possible to use the disks UUID or would this have to be implemented?

I think this would be much better for reproducibility and security.

**Comment by Lassulus** [comment]
well, disk UUIDs target partitions. this tool creates partitions.

But you should be able to use `/dev/disk/by-id` since they can also point to disks. code should look something like:
```
  cfg = {
    type = "devices";
    content = {
      "disk/by-id/wwn-0x50025388500af37d" = {
        type = "table";
        format = "msdos";
        partitions = [{
  ...
```

**Comment by Baitinq** [comment]
@Lassulus thank you so much! I didn't even know about that Linux functionality

**Comment by Lassulus** [comment]
I actually lied, the syntax I used in my example is not possible right now, but will be after I finished this PR https://github.com/nix-community/disko/pull/15

**Comment by Baitinq** [comment]
Solved by #15
